2-mandatory/2-currency-code-lookup.js

```diff
@@ -17,10 +17,38 @@ const COUNTRY_CURRENCY_CODES = [
   ["MX", "MXN"],
 ];
 
+// function createLookup(countryCurrencyCodes) {
+//   let obj = {};
+//   for (let i = 0; i < countryCurrencyCodes.length; i++) {
+//     for (let j = 0; j < countryCurrencyCodes[i].length; j++) {
+//       if (countryCurrencyCodes[i].indexOf(countryCurrencyCodes[i][j]) === 0) {
+//         obj[countryCurrencyCodes[i][j]] = countryCurrencyCodes[i][1];
+//       }
+//     }
+//   }
+//   return obj;
+// }
+//💫💫💫💫💫💫💫💫💫💫💫💫
+// function createLookup(countryCurrencyCodes) {
+//   let obj = {};
+//   for (let i = 0; i < countryCurrencyCodes.length; i++) {
+//     obj[countryCurrencyCodes[i][0]] = countryCurrencyCodes[i][1];
+//   }
+//   return obj;
+// }
+//💫💫💫💫💫💫💫💫💫💫💫💫
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let obj = {};
+  for (let i = 0; i < countryCurrencyCodes.length; i++) {
+    let arr = countryCurrencyCodes[i].values();
+    // console.log(arr.next().value);
+    obj[arr.next().value] = arr.next().value;
```

Comment:
This works well, but it isn't that easy to understand. You could rename some of the variables to make it clearer what is happening, or think of an alternative method - perhaps ask other trainees what they have done.